plugins/TKLiveSync/TKLiveSync/TKLiveSync.m

```diff
@@ -16,67 +19,172 @@ static void tryExtractLiveSyncArchive()
     NSString* libraryPath = [NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, YES) firstObject];
     NSString* liveSyncPath = [NSString pathWithComponents:@[ libraryPath, @"Application Support", @"LiveSync" ]];
     NSString* syncZipPath = [NSString pathWithComponents:@[ liveSyncPath, @"sync.zip" ]];
-
-    if (![fileManager fileExistsAtPath:syncZipPath]) {
-        return;
-    }
+    NSString* appPath = [NSString pathWithComponents:@[ liveSyncPath, @"app" ]];
 
     NSError* err;
-
-    NSString* appPath = [NSString pathWithComponents:@[ liveSyncPath, @"app" ]];
-    if ([fileManager fileExistsAtPath:appPath]) {
-        [fileManager removeItemAtPath:appPath error:&err];
+    
+    if ([fileManager fileExistsAtPath:syncZipPath]) {
```

Comment:
I haven't changed the spacing but I removed a single `!` before `[fileManager fileExistsAtPath:syncZipPath]` (not sure if I understood you correctly).